examples/binary.js

```diff
@@ -0,0 +1,235 @@
+const Launchpad = require( 'launchpad-mini' );
+// import Launchpad from 'launchpad-mini';
+
+let pad = new Launchpad();
+
+let binary = '00000001';
+//  index     12345678
+
+pad.connect().then( () => {
+    pad.reset();
+    pad.on( 'key', k => {
+        if (k.pressed) {
+            if (k.y === 7) { // bottom row
+                if (k.x !== 8) { // exclude scene buttons
+                    // let binaryDigit = 8 - k.x; // left = 8, ... , right = 1
+                    // let binaryArrayKey = 8 - binaryDigit;
+
+                    let index = k.x;
+                    let is = binary.charAt(index);
+                    let bool = is === '1';
+
+                    binary = binary.replace(/./g, (c, i) => i === index ? (bool ? 0 : 1): c); // replace binary bit with opposite
+
+                    pad.reset();
+                    drawBinary();
+                    drawNumber();
+                }
+            }
+        }
+    } );
+    drawBinary();
+    drawNumber(1);
+} );
+
+function drawBinary() {
+    let i = 0;
+    for (let char of binary) {
+        pad.col(char === '1' ? pad.green : pad.red, [ [i++, 7] ]);
```

Comment:
What would be the purpose of a low brightness red?

Reply:
I find it a bit clearer that red is “off” in binary terms. At full brightness, it does not look so much like a disabled bit.

Reply:
Well red does technically not mean off, it means a binary zero, just like green means a binary one.

I personally would prefer to keep them on the same brightness, since they 'are the same'.

Therefore i think it best to keep the brightnesses as it is, since binary has 2 states, which are equally important.

Reply:
As you wish :)